drivers/hmis/app/models/hmis/form/definition_validator.rb

```diff
@@ -172,6 +172,53 @@ def check_mutually_exclusive_attributes(document)
     link_check.call(document)
   end
 
+  def check_condition(condition, item_hash, link_id)
```

Comment:
I added a separate method `check_condition`, instead of adding this in `check_references` as suggested by the ticket, because I noticed a related todo in `check_mutually_exclusive_attributes`. I thought the additional method `check_condition` gives us a future framework for where to put all validations on conditional elements, instead of overloading `check_references`.